network-api/networkapi/utility/faker/helpers.py

```diff
@@ -45,12 +45,21 @@ def get_homepage(will_generate=False):
         raise ex
 
 
-def get_random_objects(model, max_count=5):
+def get_random_objects(model, max_count=0, exact_count=0):
     """
-    Return random objects up to a maximum number.
+    Return random objects.
```

Comment:
Kudos! This is a great change to the code base and I can see this being very helpful in the future